target/json_publication.js

```diff
@@ -313,7 +313,7 @@ var publicationJsonData={
       "unit_id": 4,
       "tonal_type_id": 1,
       "source": 1,
-      "ratio": 45.636,
```

Comment:
Hello @rnllv - could you `seed` the random parser please, so the randomised rations don't change each time you re-generate the content?

Reply:
>See here: https://www.w3schools.com/python/ref_random_seed.asp

Thanks for sharing, @IanMayo. Will implement it.